### PR TITLE
Fix amp4test request-bank

### PR DESCRIPTION
**Pull request by lannka**
PR #19118 introduced a bug in request-bank related test.  Only URL params were used as key.

### CODE REVIEW

File: build-system/amp4test.js [hunk 1]
```diff
@@ -85,8 +85,10 @@ const bank = {};
  * Deposit a request. An ID has to be specified. Will override previous request
  * if the same ID already exists.
  */
-app.use('/request-bank/deposit/*', (req, res) => {
-  const key = req.url.replace('/request-bank/deposit', '');
+app.use('/request-bank/deposit/', (req, res) => {
+  // req.url is relative to the path specified in app.use
+  const key = req.url;
+  console.log('SERVER-LOG [DEPOSIT]: ', key);
```
Comment (rsimha) [inline build-system/amp4test.js]: @lannka This line and line 108 are polluting our test logs. See https://travis-ci.org/ampproject/amphtml/jobs/451434376#L665

Can you log this using something other than `console.log`, that's silenced during tests?

Comment (lannka) [reply build-system/amp4test.js]: ah, sorry about that. The log is only helpful in local dev mode. Is there a way to have logs can be turned on/off by a flag?

Comment (rsimha) [reply build-system/amp4test.js]: Can you use the existing `quiet` flag in `gulp serve`?

Comment (lannka) [reply build-system/amp4test.js]: I want to keep those logs in local testing. I would expect a flag like : `gulp test --verbal` ?

Comment (rsimha) [reply build-system/amp4test.js]: Sent out a fix with #19329

**Pull request merged**